js/piece.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Abstract Piece class
+ */
+class Piece {
+    location = "";
+    active = false;
+    taken = false;
+    imgPath = "";
+    playerOwner = -1;
```

Comment:
These may need to be moved inside the constructor with the `this` keyword if ES6 does not yet support the public class member syntax.

Reply:
Resolved, thank you!